INSTALLATION.md

```diff
@@ -31,11 +31,11 @@ pip3 install flict
 
 ### Windows
 
-Please [help](https://github.com/vinland-technology/flict/issues/26) us writing instructions for this
+Please help us writing instructions for this
```

Comment:
The instructions are basically the same for Windows, Unix and MacOS, so I kinda think this can be merged into a one liner `pip3 install flict`